.travis/script.sh

```diff
@@ -1,21 +1,38 @@
 #!/usr/bin/env bash
 # coding=utf-8
 
-set -veuo pipefail
+set -mveuo pipefail
+
+wait_for_pulp() {
+  TIMEOUT=${1:-5}
+  while [ "$TIMEOUT" -gt 0 ]
+  do
+    echo -n .
+    sleep 1
+    TIMEOUT=$(($TIMEOUT - 1))
+    if [ $(http :8000/pulp/api/v3/status/ | jq '.database_connection.connected and .redis_connection.connected') = 'true' ]
+    then
+      echo
+      return
+    fi
+  done
+  echo
+  return 1
+}
 
 if [ "$TEST" = 'docs' ]; then
+  pulp-manager runserver >> ~/django_runserver.log 2>&1 &
+  sleep 5
```

Comment:
what about using wait_for_pulp here too? We've had issues in Travis runs with this exact issue but we never had `wait_for_pulp`!

Reply:
Yes, there is room for improvement: In it's actual state `wait_for_pulp` checks the availability of the worker threads (and could also check the content app if it were present in the `status` endpoint). So, it is not usable here unmodified.

Reply:
OK we can leave this as is for now